Server/Server/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for Server project.
+
+Generated by 'django-admin startproject' using Django 4.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-_w=guh&@n^%ydou=0%47^!r^n-bz91se^b9cg&52xi3wnpi!9c"
```

Comment:
I don't know if this is the actual secret key we will be using. Looks like an auto-generated django key. this should probably be replaced and also we should not keep the secret key in the public repo.

See how you would use an load the secret key from an environment variable instead, or however django suggests to do this in production. It looks like there is a comment above this that goes over that